tests/Integration/WPThemeTestcase.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration;
+
+use WP_Theme;
```

Comment:
```suggestion
use WP_Theme;
use WP_Rocket\Tests\Integration\FilesystemTestCase;
```

Reply:
Turns out we don't need the `use` here -- they are in the same namespace :)